Tools/CISupport/ews-build/factories.py

```diff
@@ -97,6 +97,27 @@ def __init__(self, platform, configuration=None, architectures=None, triggers=No
         self.addStep(ApplyWatchList())
 
 
+class SmartPointerStaticAnalyzerFactory(factory.BuildFactory):
```

Comment:
Please add unit-test for this Factory (in `factories_unittest.py`)

Reply:
I will add this when I made the `config.json` changes to add a builder. If I add the unit-test for the factory without specifying a builder, it will error with `AssertionError: False is not true : Builder Smart-Pointer-Static-Analyzer doesn't exist, but has unnecessary expected steps`